WellenceChallenge/WellenceApp/services.py

```diff
@@ -0,0 +1,43 @@
+from .models import Tasks
+from django.shortcuts import get_object_or_404
+
+class TaskService:
+    @staticmethod
+    def create_task(task_data):
```

Comment:
Always try to specify a type when possible to allow your IDE to give you better typing suggestions and avoid mistakes.

So do `def create_task(task_data: TaskIn)` instead of this,
Even `def create_task(task_data: Dict[str, Any])` is better than nothing.